themes/aura/styles/thankyou.scss

```diff
@@ -150,15 +150,19 @@
         border-bottom: 1px solid #EBEBEB;
 
         >* {
-          text-align: start;
+          text-align: var(--yc-theme-direction);
```

Comment:
This would map out to `text-algin: rtl | ltr` which are not. valid text-align properties

Reply:
I thought it have `left | right`